lib/compilers/pascal.ts

```diff
@@ -88,25 +98,34 @@ export class FPCCompiler extends BaseCompiler {
         return options;
     }
 
-    getOutputFilename(dirPath) {
+    override getOutputFilename(dirPath: string) {
         return path.join(dirPath, `${path.basename(this.compileFilename, this.lang.extensions[0])}.s`);
     }
 
-    getExecutableFilename(dirPath) {
+    override getExecutableFilename(dirPath: string) {
         return path.join(dirPath, 'prog');
     }
 
-    async objdump(outputFilename, result, maxSize, intelAsm, demangle) {
+    override async objdump(
+        outputFilename,
+        result: any,
+        maxSize: number,
+        intelAsm,
+        demangle,
+        staticReloc: boolean,
+        dynamicReloc: boolean,
+        filters: ParseFiltersAndOutputOptions,
+    ) {
         const dirPath = path.dirname(outputFilename);
         const execBinary = this.getExecutableFilename(dirPath);
         if (await utils.fileExists(execBinary)) {
-            return super.objdump(execBinary, result, maxSize, intelAsm, demangle);
+            return super.objdump(execBinary, result, maxSize, intelAsm, demangle, staticReloc, dynamicReloc, filters);
```

Comment:
Same here, three arguments weren't being passed and also weren't present in the parameter list. I went ahead and just added them to the parameters / forwarded.